README.md

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
This file still contains instructions for `hs_hub` but fixing this isn't urgent tbh